### ISSUE TITLE
Dar llave a Mar

### ISSUE THREAD

**Issue by Jaimee007** [asrob-uc3m/actas#171]
Al no tener tiempo entre las reuniones y las clases sugerimos tener una llave para no depender de nadie cuando vayamos antes de la hora de inicio.

**Comment by madelgj** [comment]
Sí, que siempre llegamos antes a robofactory y ya que me dejáis la responsabilidad de ser secretaria, creo que puedo asumir está también

**Comment by JorFru** [comment]
Creo que habría que sugerir un cambio en los estatutos y su posterior votación.
El cambio sugerido podría ser:
Personas con derecho a poseer llave: secretario.
Actualmente creo que sólo pueden tenerla los jefes de grupo y el presidente con el manojo único.

**Comment by GioSunrider** [comment]
Cada grupo de trabajo tiene permitidas dos llaves. Igual se le podria asignar la segunda de Robofactory, que creo que actualmente solo tiene @David-Estevez

**Comment by jgvictores** [comment]
En la issue correspondiente https://github.com/asrob-uc3m/estatutos/issues/18 ([aquí](https://github.com/asrob-uc3m/estatutos/issues/18#issuecomment-383389172)) comenté que:
1. No hay norma escrita en los estatutos como tal
1. Lo más antiguo que tenemos documentado es https://github.com/asrob-uc3m/actas/issues/15, donde se habla de 2 llaves por grupo (que fue en las fechas en las que colocamos el candado que debe ser el actual)

Totalmente de acuerdo con avanzar en https://github.com/asrob-uc3m/estatutos/issues/18, votaciones etc. Lo de los 2 responsables es conveniente justo porque casos como el de esta issue ocurren asiduamente.

**Comment by GioSunrider** [comment]
Fue el mismo caso por el que el año pasado recibí las llaves por printer.

**Comment by David-Estevez** [comment]
Al leer el asunto pensé que el problema era darle una llave a Mar :rofl: 

En primer lugar, creo que la issue debería ser "Administración de llaves por parte de junta" o "Administración de llaves por parte de los grupos activos", porque en todo caso se le daría las llaves a un cargo, no a una persona.

En segundo lugar, hay que especificar de qué llave habla esta issue, para evitar confusiones: nave o armario.

En tercer lugar,
* Si la issue es "Administración de llaves por parte de junta" , deberíamos pensar una motivación para que la junta tenga llaves. 
* Si la issue es "Administración de llaves por parte de los grupos activos", la motivación está clara, y símplemente habría que regularlo y buscar una forma de documentar quién tiene llave en el momento de recibirla, de cara a que se puedan reclamar de vuelta / localizar en el futuro.

PD:
> Cada grupo de trabajo tiene permitidas dos llaves. Igual se le podria asignar la segunda de Robofactory, que creo que actualmente solo tiene @David-Estevez

Yo tengo llave del armario a través de ASROB, pero mi llave de la nave es independiente y anterior a ASROB. Por eso decía que conviene especificar de qué llave se está hablando.

**Comment by JorFru** [comment]
Parece que está claro el tema de crear un documento que asocie cada grupo de trabajo / responsable / llaves que posee.

Cuando lo tenga lo pongo por aquí.

Esto viene de https://github.com/asrob-uc3m/estatutos/issues/18

**Comment by JorFru** [comment]
Veo claro que es más fácil que Mar consiga las llaves (nave y armario) por su condición de participar en RoboFactory que de ser secretaria.
